Libraries/Alert/Alert.js

```diff
@@ -72,7 +72,16 @@ type Options = {
  * ```
  */
 class Alert {
-
+  /**
+   * Displays an alert dialog
+   *
+   * Options is an object that accepts `cancelable`. If true,
+   * the modal will not close by pressing the background, only
+   * by pressing a button.
+   *
+   * Example:
+   * `{cancelable: true}`
```

Comment:
From the docs above (line 55) it looks like this is actually the opposite? When `false`, the modal can't be closed by tapping outside.